package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Module_5",
+  "version": "1.0.0",
+  "description": "For this assignment, you're going to fork this repository to your own Github account and clone that repository locally. Inside the repo you'll find a placeholder for app.js, a placeholder routes/truckRoutes.js, a model/truckModel.js, and a 'public' directory. Inside the public directory you'll find an index.html file, a trucks.css file, and a client.js file.",
+  "main": "app.js",
+  "dependencies": {
+    "body-parser": "^1.12.4",
+    "express": "^4.12.3",
+    "mongoose": "^4.0.3",
+    "nodemon": "^1.3.7",
```

Comment:
When you install nodemon, I would avoid installing it as a dependency for your project as it's not part of the code you would deploy to production. This is just a tool a developer chooses to use or not locally.